.ci/jenkins_job_build.sh

```diff
@@ -48,10 +48,11 @@ then
 	go get github.com/mattn/goveralls
 fi
 
-# Get the repository and move to the correct commit
-go get ${cc_repo} || true
+# Get or update the repository, but do build ie.
```

Comment:
Nit: should that be "but do not build it."?

Reply:
Indeed. Some odd finger/brain trouble wen't on there.... fixed.